### PR TITLE
[s2t]  streaming conformer u2 and u2pp jit export

### PR DESCRIPTION
**Pull request by zh794390558**
### PR types

New features
### PR changes

Others
### Describe

u2pp/u2 support static export
decoder using `fowrad_attention_decoder` to replace `decoder`
remove useless cast for bool operations
reverse_weight in decode config file
fix audio fbank api for static export

### CODE REVIEW

File: paddlespeech/s2t/modules/attention.py [hunk 1]
```diff
@@ -73,9 +84,14 @@ def forward_qkv(self,
...
         """
         n_batch = query.shape[0]
+
         q = self.linear_q(query).view(n_batch, -1, self.h, self.d_k)
-        k = self.linear_k(key).view(n_batch, -1, self.h, self.d_k)
-        v = self.linear_v(value).view(n_batch, -1, self.h, self.d_k)
+        # k = self.linear_k(key).view(n_batch, -1, self.h, self.d_k)
+        # v = self.linear_v(value).view(n_batch, -1, self.h, self.d_k)
+        k, v = F.linear(key, self.weight, self.bias).view(
+            n_batch, -1, 2 * self.h, self.d_k).split(
```
Comment (Zth9730) [inline paddlespeech/s2t/modules/attention.py]: if self.training:
            k = self.linear_k(key).view(n_batch, -1, self.h, self.d_k)
            v = self.linear_v(value).view(n_batch, -1, self.h, self.d_k)
        else:
            k, v = F.linear(key, self.weight, self.bias).view(
                n_batch, -1, 2 * self.h, self.d_k).split(
                    2, axis=2)

**Review by Zth9730 — APPROVED** [review]
LGTM
